### ISSUE TITLE
slither-check-upgradeability add `sither.config.json`

### ISSUE THREAD

**Issue by wiasliaw** [crytic/slither#1260]
### Describe the desired feature

IDK `slither` can use config in `slither.config.json`, but tools like `slither-check-upgradeability` have to add remappings in command.

**Comment by enzoevers** [comment]
I agree that it would be very useful to make `slither.config.json` work for the other tools too.
